site/content/en/docs/concepts/topology_aware_scheduling.md

```diff
@@ -117,6 +117,20 @@ Check also [PodSet updates in ProvisioningRequestConfig](site/content/en/docs/ad
 to see how you can configure Kueue if you want to restrict scheduling to the
 newly provisioned nodes (assuming the provisioning class supports it).
 
+### Hot swap support
```

Comment:
please also add a note in this section that since finding replacement node within the old domain (like rack) may not be possible we recommend using waitForPodsReady.recoveryTimeout, to prevent workload waiting for the replacement indefinetly